lib/streamlit/delta_generator.py

```diff
@@ -355,16 +356,41 @@ def wrapper(*args: Any, **kwargs: Any) -> NoReturn:
         return wrapper
 
     @property
-    def _parent_block_types(self) -> Iterable[str]:
+    def _parent_block_types(self) -> Iterable[Tuple[str, Optional[int]]]:
```

Comment:
nit: I'd introduce type aliases, something like:

```
BlockType = str
RootContainer = int
PerentBlockTypes = Tuple[BlockType, Optional[RootContainer]]
```

Reply:
Good idea, but since `RootContainer` is reserved I've just used `int` here 👍